xrviz/templates/variable.html

```diff
@@ -112,8 +112,9 @@
 
         div.table_div {
           font-size: 13px;
-          overflow: scroll; 
-          height: 140px;
+          overflow: scroll;
```

Comment:
This style will surely change, don't worry too much. That you need to tinker with absolute point-measured like this i a bad indication that it won't display well in a range of resolutions. Same goes for the Panel widget sizes elsewhere.